client/state/data-layer/wpcom/sites/media/index.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Internal dependencies
+ */
+import { MEDIA_REQUEST } from 'state/action-types';
+import { isRequestingMedia } from 'state/selectors';
+import { failMediaRequest, receiveMedia, requestingMedia } from 'state/media/actions';
+import wpcom from 'lib/wp';
+import debug from 'debug';
+
+/**
+ * Module variables
+ */
+const log = debug( 'calypso:middleware-media' );
+
+/**
+ * Issues an API request to fetch media for a site and query.
+ *
+ * @param  {Object}  store  Redux store
+ * @param  {Object}  action Action object
+ * @return {Promise}        Promise
```

Comment:
Leaving it off will give an eslint warning, which is why I added it here. Similarly "Action object" for `{Object}  action` is somewhat redundant too. :)